BUCK

```diff
@@ -2,7 +2,7 @@
 genrule(
     name = '_bundle-javascript',
     srcs = glob( [ 'src/main/javascript/**/*' ] ),
-    cmd = 'browserify --debug src/main/javascript/home.js > $OUT',
+    cmd = 'nodejs ./../../../node_modules/.bin/browserify --debug src/main/javascript/home.js -o $OUT',
```

Comment:
Do you have any idea about the "./../../../node_modules/.bin/browserify"? It was trying to run browserify from within the buck-out directory (can't remember the full path), and I couldn't work out where buck was getting that from.